packages/babel-plugin/src/__tests__/stubs/objects.js

```diff
@@ -29,3 +29,5 @@ const cantEvaluate = (() => 12)();
 export const cantStaticallyEvaluate = {
   fontSize: cantEvaluate,
```

Comment:
didn't we say we had to keep evaluating IFEEs?

Reply:
In this PR I am only checking the **function type node** and recursively calling the same function where we currently have check for `identifier`, `memberExpression` and now `function`. Earlier I had check for `callExpression` which I removed coz it was going beyond scope of this PR. What you think? Should we cover `callExpression` part later for cases like `const cantEvaluate = (() => 12)();`